### PR TITLE
Update to Liferea 1.13.9

### PR DESCRIPTION
**Pull request by bbhtt**

### CODE REVIEW

File: 0001-Update-appdata-to-1.13.9.patch [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
 diff --git a/net.sourceforge.liferea.appdata.xml.in b/net.sourceforge.liferea.appdata.xml.in
-index cc9b7cf8..cedc7e0e 100644
+index cc9b7cf8..c8b1ca6e 100644
 --- a/net.sourceforge.liferea.appdata.xml.in
 +++ b/net.sourceforge.liferea.appdata.xml.in
 @@ -67,6 +67,7 @@
      <content_attribute id="money-gambling">none</content_attribute>
    </content_rating>
    <releases>
-+    <release version="1.13.8"  date="2022-04-05" />
```
Comment (mikelolasagasti) [inline 0001-Update-appdata-to-1.13.9.patch:18]: Patch should include both versions

Comment (bbhtt) [reply 0001-Update-appdata-to-1.13.9.patch:18]: Ah yea, thanks!

**Comment by bbhtt** [comment]
Hey, @mikelolasagasti the build from the merge commit is failing https://github.com/flathub/net.sourceforge.liferea/commit/dfc16240f09e947bc4df9c4a2325b1c6bddd5f43 so it won't get published. Please retrigger it. Thanks!

**Comment by mikelolasagasti** [comment]
bot, build

**Comment by mikelolasagasti** [comment]
> Hey, @mikelolasagasti the build from the merge commit is failing [dfc1624](https://github.com/flathub/net.sourceforge.liferea/commit/dfc16240f09e947bc4df9c4a2325b1c6bddd5f43), https://buildbot.flathub.org/#/builders/39/builds/12335 so it won't get published. Please retrigger it. Thanks!

Should be fine now: https://buildbot.flathub.org/#/builders/29/builds/56179

**Pull request merged**